### ISSUE TITLE
Doesn't seem to work on Unsplash

### ISSUE THREAD

**Issue by windyweather** [belaviyo/save-images#42]
Can't get your extesnions to work on this page:

https://unsplash.com/@jayson_hinrichsen

Sure would like to get images from Unsplash. But size restrictions appear to be ignored and not all the images from the page appear in the download. Even if I make the time very long 2 min, and the other restrictions very large. Bummer.

**Comment by belaviyo** [comment]
It works for me.
![Screen Shot 2020-05-15 at 7 51 17 AM](https://user-images.githubusercontent.com/1221138/82016342-5bddfd80-9670-11ea-9e3c-b02a73e237ee.png)

The extension was able to get both small and large sizes at level one search. The issue is the extension does not know the dimension of the large image since it is not in the current document. You just need to download all images or restrict only by image sizes.

**Comment by windyweather** [comment]
Ok. But I'm not clear how you got a different result than I did. I only got about 66 of the 109 photos that the page says it has. And a manual count gets about 111, which is approximately correct. Here's what I got. What am I don't wrong?
Thanks.

![ManualCountOfPhotos](https://user-images.githubusercontent.com/4505159/82067017-0233fe00-9685-11ea-8ca0-74f10734dfe3.png)
![ArchivePhotos](https://user-images.githubusercontent.com/4505159/82067020-02cc9480-9685-11ea-89e1-bac974acc77b.png)
![CaptureDialog](https://user-images.githubusercontent.com/4505159/82067025-03652b00-9685-11ea-965c-2eb808f497a8.png)
